src/main/java/core/basesyntax/validator/quantity/QuantityValidatorImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.validator.quantity;
+
+public class QuantityValidatorImpl implements QuantityValidator {
+    @Override
+    public boolean isQuantityCorrectForPurcase(long quantity,
+                                               long fruitBalace, int lineNumber) {
```

Comment:
```suggestion
    public boolean isQuantityCorrectForPurchase(long quantity, long fruitBalance) {
```
it is redundant to pass lineNumber in your case